Automattic/jetpack
# Search: Always show save button for mobile Customberg

Fixes #20908.

#### Changes proposed in this Pull Request:
- Refactors styles to always show the save button for all mobile headers.
- Changes labeling for the mobile sidebar button so that the "close sidebar" button is now a "show preview" button.

<table>
<tr>
	<td><img width="506" alt="Screen Shot 2021-09-28 at 3 55 30 PM" src="https://user-images.githubusercontent.com/4044428/135171155-a54dcf90-7791-441b-8d1f-a7b618335b6f.png">
	<td><img width="506" alt="Screen Shot 2021-09-28 at 3 55 33 PM" src="https://user-images.githubusercontent.com/4044428/135171151-7d21b241-7480-47bb-9840-e5019df9680e.png">
<tr>
<td>Initial interface for mobile users
<td>Interface show after clicking the white image button
</table>

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your Jetpack site with a Search subscription.
* Navigate to `/wp-admin/admin.php?page=jetpack-search-configure` on a mobile viewport.
* Ensure that the Save button is always shown in the mobile header.
